### ISSUE TITLE
Turkish character problem when listing pages

### ISSUE THREAD

**Issue by Emperyal** [gturri/nspages#102]
First thanks for this great plugin.

I have a small problem. I'm using the nspages plugin to sort and group pages alphabetically by letter. But it seems to put the pages that start with the Turkish character (Ç, Ş, Ğ, İ, Ö, Ü) at the end as shown below:

![ss_dokuwiki](https://user-images.githubusercontent.com/46525068/75866777-9fb5ba80-5e16-11ea-931c-947cede61699.png)

How can I fix this?

**Comment by gturri** [comment]
Hi,

Just to be sure I understand correctly: do you mean that those characters should be first?

About the way ordering is done: default sorting relies on the php `strcasecmp` function (used on https://github.com/gturri/nspages/blob/14684cc8698b3984d1a31e120c3d65b3904cc37f/printers/printer.php#L95 ).

When I run this PHP function I see the following results:

```
php > echo strcmp('a', 'b'); 
-1                           
php > echo strcmp('a', 'c');
-2                           
php > echo strcmp('a', 'd'); 
-3                           
php > echo strcmp('d', 'd'); 
0                            
php > echo strcmp('d', 'a'); 
3                            
php > echo strcmp('A', 'Ç'); 
-130                         
php > echo strcmp('A', 'Ş'); 
-132
```

so the nspages behavior seems consistent with the behavior of the php function.

Do you mean that another function would be more relevant here?

**Comment by Emperyal** [comment]
Sorry, I don't understand PHP functions much. I mean those characters should follow the Turkish alphabetical order. For instance, the letter Ç in the screenshot above should be preceded by the letter B and İ should be preceded by I. But such characters are sorted out of the alphabetical order.

Anyway, I solved my problem by using `-dictionaryOrder` option with the local name `turkish.utf8` and pages are listed in the alphabetical order I wanted. I had tried this option with other local names like tr, TR, tr_TR etc. but I couldn't get a result.  Using `-dictionaryOrder=turkish.utf8` fixed my issue.

Thank you anyway for your interest.
